Library/Formula/gnu-spell.rb

```diff
@@ -0,0 +1,23 @@
+class GnuSpell < Formula
+  desc "Unix spell emulator"
+  homepage "https://savannah.gnu.org/projects/spell/"
+  url "http://ftpmirror.gnu.org/spell/spell-1.1.tar.gz"
+  mirror "https://ftp.gnu.org/gnu/spell/spell-1.1.tar.gz"
+  sha256 "7cc243547eb31360ff769835dc88b5ba4f2f47385daa230523beecf9e1c29744"
+
+  depends_on "coreutils" => :build
+  depends_on "ispell"
+
+  def install
+    inreplace "Makefile.in", "$(prefix)/info", "@infodir@"
```

Comment:
Just so we're all aware, this has been released 1997, and was last updated
2011 (see http://ftp.gnu.org/gnu/spell/ ).

Ping @cateee

On Mon, Nov 30, 2015 at 9:27 AM, Mike McQuaid <notifications@github.com>
wrote:

> In Library/Formula/gnu-spell.rb
> <https://github.com/Homebrew/homebrew/pull/46347#discussion_r46117302>:
>
> > @@ -0,0 +1,23 @@
> > +class GnuSpell < Formula
> > +  desc "Unix spell emulator"
> > +  homepage "https://savannah.gnu.org/projects/spell/"
> > +  url "http://ftpmirror.gnu.org/spell/spell-1.1.tar.gz"
> > +  mirror "https://ftp.gnu.org/gnu/spell/spell-1.1.tar.gz"
> > +  sha256 "7cc243547eb31360ff769835dc88b5ba4f2f47385daa230523beecf9e1c29744"
> > +
> > +  depends_on "coreutils" => :build
> > +  depends_on "ispell"
> > +
> > +  def install
> > +    inreplace "Makefile.in", "$(prefix)/info", "@infodir@"
>
> Also reporting this upstream would be nice [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/46347/files#r46117302>.
>



-- 
andreineculau.com <http://www.andreineculau.com>